TwilioVerifyDemo/TwilioVerifyDemo.xcodeproj/project.pbxproj

```diff
@@ -590,10 +590,10 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_ENTITLEMENTS = TwilioVerifyDemo/TwilioVerifyDemo.entitlements;
-				CODE_SIGN_IDENTITY = "iPhone Developer";
+				CODE_SIGN_IDENTITY = "iPhone Distribution";
 				CODE_SIGN_STYLE = Manual;
 				CURRENT_PROJECT_VERSION = 1;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = 9EVH78F4V4;
```

Comment:
The development team can change for someone running the sample app, can we keep it empty?

Reply:
Can we set it when generating the app? I am not sure about including a specific provisioning profile (it will be different for developers) or the team id

Reply:
Done